server/app.js

```diff
@@ -208,12 +208,22 @@ module.exports = function createApp({
       user: req.user,
       currentUrlPath: req.baseUrl + req.path,
       hostname: req.hostname,
+      authUrl: config.nomis.authUrl,
+      apiClientId: config.nomis.apiClientId,
     }
     next()
   }
 
   app.use(addTemplateVariables)
 
+  function addreturnURL(req, res, next) {
```

Comment:
super meh.. but `R` for `return`

Reply:
pushing up a correction for this now - thought I might as well.